src/data/eth2-articles.js

```diff
@@ -25,21 +25,25 @@ export const benArticles = [
   {
     title: "What’s New in Eth2 – #82",
     description: "19 November 2021",
-    link: "https://hackmd.io/@benjaminion/eth2_news/https%3A%2F%2Fhackmd.io%2F%40benjaminion%2Fwnie2_211119",
```

Comment:
@vluna Not sure why but your linter has went a bit crazy here. Any chance we could remove the files that aren't concerned with fixing the stacking elements from this PR?

Reply:
@minimalsm I fixed most of the files, just a couple not sure why I could not revert them back. Those couple of files are just white spaces not sure if that would be a problem? If it is, I will just create a new PR with the `/stablecoins` change.